pkgs/top-level/php-packages.nix

```
@@ -311,10 +311,8 @@ lib.makeScope pkgs.newScope (self: with self; {
         }
         {
           name = "enchant";
-          buildInputs = [ enchant1 ];
-          configureFlags = [ "--with-enchant=${enchant1}" ];
-          # enchant1 doesn't build on darwin.
-          enable = (!stdenv.isDarwin);
+          buildInputs = [ enchant2 ];
+          configureFlags = [ "--with-enchant" ];
           doCheck = false;
```

While at it, we can enable tests

```suggestion
          preCheck = ''
            export XDG_DATA_DIRS="${hunspellDicts.en-us}/share"
          '';
```